.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Updated this because github actions was taking very long to find a runner for 18.04. Updating this made the actions run instantly